bin/coding-game-service.js.in

```diff
@@ -131,9 +131,11 @@ const CodingGameServiceApplication = new Lang.Class({
 
     vfunc_dbus_register: function(conn, object_path) {
         this.parent(conn, object_path);
-        let logFile = Gio.File.new_for_path(GLib.build_filenamev([GLib.get_user_config_dir(),
-                                                                  'com.endlessm.CodingGameService',
-                                                                  'game-service.log']));
+        let logFile = Gio.File.new_for_path(GLib.build_filenamev([
```

Comment:
That probably could be squashed with the `bin: Use correct indentation for multiline arrays` commit.